test/lotus_setup_test.rb

```diff
@@ -11,13 +11,14 @@
         Bundler = Module.new do
           extend self
 
-          def require(groups)
+          def require(default, groups)
```

Comment:
I don't see where `default` group get merged into `groups`, what do you think about reusing the API interface of the original [Bundler.require](https://github.com/bundler/bundler/blob/master/lib/bundler.rb#L168-L170) instead?